src/ovirtapi/transform.test.js

```diff
@@ -0,0 +1,343 @@
+/* eslint-env jest */
+
+import { VmStatistics } from './transform'
+
+const testData = {
+  title: 'test vm statistics',
+  test: [
```

Comment:
Please break each test into its own test case so they can fail independently.

You can make the `describe` a loop.  Here is an example:
https://github.com/oVirt/ovirt-web-ui/blob/a151047be41d90d29c40fde5d7b532ffd5e46729/src/intl/messages.test.js#L53-L75

Basically the `describe.each()` call runs 1 test per iterator item.  Make your test data look more like:
```jsx
const TEST_DATA = [
  {
    title: 'test 1',
    test: [
      ...
    ],
    expect: [
    ]
  },
  {
    title: 'test 2',
    test: ...,
    expect: ...,
  },
  ...
]
```

Reply:
Done.